tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -118,12 +118,12 @@ def test_tokenizer_integration(self):
 
     def test_output_offsets(self):
         tokenizer = self.get_tokenizer()
-        previous_sequence = [51492, 406, 3163, 1953, 466, 13, 51612, 51612]
+        previous_sequence = [51492, 406, 3163, 1953, 466, 13, 51612]
         self.assertEqual(
             tokenizer.decode(previous_sequence, output_offsets=True),
             {
-                "text": " not worth thinking about.",
-                "offsets": [{"text": " not worth thinking about.", "timestamp": (22.56, 24.96)}],
+                "text": "<|22.56|> not worth thinking about.<|24.96|>",
```

Comment:
Timestamp tokens are now treaded as added special tokens, so they appear in the decoded output

Reply:
They don't have to no? You can do both. Return timestamps or return in the text.

Reply:
Hmm we can't skip the timestamp tokens from the decoded text - they're not actually treated as special tokens, just regular added tokens:
```python
tokenizer.decode(previoius_sequence, skip_special_tokens=True)
```
**Output:**
```
'<|22.56|> not worth thinking about.<|24.96|>'
```

Should we update the script in https://github.com/huggingface/transformers/pull/24476#issue-1773763792 to add the timestamp tokens as **special** token ids? Rather than just added tokens?